enterprise/frontend/src/metabase-enterprise/upload_management/DeleteConfirmModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { c, t } from "ttag";
+
+import { Box, Flex, Text, Button, Modal, Switch } from "metabase/ui";
+import type { Table } from "metabase-types/api";
+
+export function DeleteConfirmModal({
+  opened,
+  tables,
+  onConfirm,
+  onClose,
+}: {
+  opened: boolean;
+  tables: Table[];
+  onConfirm: (sendToTrash: boolean) => void;
+  onClose: () => void;
+}) {
+  const [sendToTrash, setSendToTrash] = useState(true);
+  const title =
+    tables.length === 1
```

Comment:
my inner Raffi voice says something about using `ngettext` and `msgid` here instead :shrug:

Reply:
I don't think ngettext works with the context function. so @rafpaf can have one or the other here.

Reply:
Idea: on lines 23-24, use `c("...").ngettext`, covering both the singular and plural cases. Then the translators can translate the plural cases properly. I don't know if POEditor lets you avoid providing the singular case. I realize it will never be used.

Reply:
@iethree I can't tell if you resolved the thread before or after I posted my reply 😁 . I've re-opened it but of course feel free to re-resolve it.

Reply:
I don't see a benefit here in adding an additional unused translation string. 😕